site/template.html

```diff
@@ -12,5 +12,7 @@
 <body>
 {content}
 </body>
+<script src='/assembly/clipboard.min.js'></script>
```

Comment:
When I run `npm start` and try copying, it does not work. Something is missing ...